pnpm-workspace.yaml

```diff
@@ -0,0 +1,2 @@
+packages:
+  - "packages/*/*"
```

Comment:
Adds optional support for pnpm workspaces, this is how I found all the phantom dependencies.

I can remove this (or add it to a `./scripts/setup-pnpm.bash` script for those who want to opt-in) if it's too contentious though